src/commands/registerCommands.ts

```diff
@@ -3,49 +3,56 @@
+import { commands, TextDocument, workspace } from 'vscode';
+import { AppSettingTreeItem, FileTreeItem, registerSiteCommand } from 'vscode-azureappservice';
+import { AzureTreeItem, IActionContext, openInPortal as uiOpenInPortal, registerCommand, registerEvent } from 'vscode-azureextensionui';
+import { DeploymentSlotsNATreeItem, ScaleUpTreeItem } from '../explorer/DeploymentSlotsTreeItem';
+import { ext } from '../extensionVariables';
+import { addAppSetting } from './appSettings/addAppSetting';
+import { deleteAppSetting } from './appSettings/deleteAppSettings';
+import { downloadAppSettings } from './appSettings/downloadAppSettings';
+import { editAppSetting } from './appSettings/editAppSetting';
+import { renameAppSetting } from './appSettings/renameAppSetting';
+import { toggleSlotSetting } from './appSettings/toggleSlotSetting';
+import { uploadAppSettings } from './appSettings/uploadAppSettings';
+import { browseWebsite } from './browseWebsite';
+import { addCosmosDBConnection } from './connections/addCosmosDBConnection';
+import { removeCosmosDBConnection } from './connections/removeCosmosDBConnection';
+import { revealConnection } from './connections/revealConnection';
+import { revealConnectionInAppSettings } from './connections/revealConnectionInAppSettings';
+import { createSlot } from './createSlot';
+import { createWebApp, createWebAppAdvanced } from './createWebApp/createWebApp';
+import { deleteWebApp } from './deleteWebApp';
+import { deploy } from './deploy/deploy';
+import { deploySlot } from './deploy/deploySlot';
+import { connectToGitHub } from './deployments/connectToGitHub';
+import { disconnectRepo } from './deployments/disconnectRepo';
+import { editScmType } from './deployments/editScmType';
+import { redeployDeployment } from './deployments/redeployDeployment';
+import { viewCommitInGitHub } from './deployments/viewCommitInGitHub';
+import { viewDeploymentLogs } from './deployments/viewDeploymentLogs';
+import { generateDeploymentScript } from './generateDeploymentScript';
+import { ImportUriHandler } from './ImportUriHandler';
+import { installCosmosDBExtension } from './installCosmosDBExtension';
+import { enableFileLogging } from './logstream/enableFileLogging';
+import { startStreamingLogs } from './logstream/startStreamingLogs';
+import { stopStreamingLogs } from './logstream/stopStreamingLogs';
+import { openInPortal } from './openInPortal';
+import { startRemoteDebug } from './remoteDebug/startRemoteDebug';
+import { restartWebApp } from './restartWebApp';
+import { showFile } from './showFile';
+import { startSsh } from './startSsh';
+import { startWebApp } from './startWebApp';
+import { stopWebApp } from './stopWebApp';
+import { swapSlots } from './swapSlots';
+import { importTrialApp } from './trialApp/importTrialApp';
+import { removeTrialApp } from './trialApp/removeTrialApp';
 
 export function registerCommands(): void {
+    // tslint:disable-next-line: no-unused-expression
+    new ImportUriHandler();
```

Comment:
Could you move this to `extension.ts`?  It feels a bit out of place in `registerCommands`.